### DISCUSSION TITLE
Multiple entries on same day for cumulative habits, such as drinking water -- possible?

### DISCUSSION THREAD

**Discussion by fossy1** [iSoron/uhabits#2054] [Feature Requests]
I am working to drink 100 ounces of water per day. I drink water in approximately 8 ounce increments, and so on a typical day I will have 12 or 13 entries to track. Can Loop Habit Tracker accommodate multiple entries on the same day, and keep a running total of the amounts entered each time, and track the total against my daily goal?
Thanks for considering.

**Comment by esceral** [comment]
Seconded. This would be really useful for all sorts of nutrition and exercise habits, like “consume less than x kcal/day” or “cycle x km/day”.
